content/en/news/security/istio-security-2021-002/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: ISTIO-SECURITY-2021-002
+subtitle: Security Bulletin
+description: Upgrades from older Istio versions can affect access control to an ingress gateway due to a change of container ports.
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["All releases 1.6 and later"]
+publishdate: 2021-04-07
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+Upgrading from Istio versions 1.5 and prior, to 1.6 and later, may result in misconfigured access control:
```

Comment:
@craigbox how about this:

Upgrading from Istio versions 1.5 and prior, to 1.6 and later, may result in access control bypass:

Reply:
wfm!